v0/util/index.js

```diff
@@ -712,7 +712,7 @@ const constructPayload = (message, mappingJson, destinationName = null) => {
       if (value || value === 0 || value === false) {
         if (destKey) {
           // set the value only if correct
-          set(payload, destKey, value);
+          _.set(payload, destKey, value);
```

Comment:
What called for this change? We are already importing a set method on Ln 13.

Reply:
I have used some attributes in trackConfig.json i.e. `"destKey":"conversionAdjustments[0].restatementValue.currencyCode"`. Using the set from set-value is not working in this way, whereas set from lodash working.